cfgov/scripts/http_smoke_test.py

```diff
@@ -102,14 +102,15 @@
 SHORT_RUN = [
     '/',
     '/es/',
-    '/complaint/',
+    # ('/es/obtener-respuestas/buscar'
```

Comment:
I was just trying to get the totality of the list at parity with the `LONG_RUN` list, which is what the original intent seems to have been.